android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -557,21 +558,21 @@ public void onLinkCreate(String url, BranchError error) {
                                             .build();
                                     AppInviteDialog appInviteDialog = new AppInviteDialog(MainActivity.this);
                                     appInviteDialog.registerCallback(callbackManager, new FacebookCallback<AppInviteDialog.Result>() {
-                                @Override
-                                public void onSuccess(AppInviteDialog.Result result) {
-                                    alertDialog.cancel();
-                                }
+                                        @Override
```

Comment:
Always use (Cntrl + Alt + L) in Android Studio to reformat code in good manner. I hope it helps.